docs/telescopes/unpaywall_data_feed.md

```diff
@@ -0,0 +1,62 @@
+# Unpaywall Data Feed
+
+"Unpaywall is a project of Our Research, a nonprofit building tools to help make scholarly
+research more open, accessible, and reusable." ... they "harvest Open Access content from
+over 50,000 publishers and repositories, and make it easy to find, track, and use."
+-- [Unpaywall website](https://unpaywall.org/).
+
+Unpaywall is an "open database of free scholarly articles." It includes "data from open indexes like Crossref 
+and DOAJ where it exists." Data comes from "monitoring over 50,000 unique online content hosting locations, 
+including Gold OA journals, Hybrid journals, institutional repositories, and disciplinary repositories." 
+"Unpaywall assigns an OA Status to every article." "There are five possible values: closed, green, gold, 
+hybrid, and bronze."
+” _- source: [Unpaywall](https://unpaywall.org/)_ 
+and [data details](https://unpaywall.org/data-format)
+
+The free Unpaywall snapshot service is only updated a few times a year.  It is also difficult to find changes
+from snapshot to snapshot. The Data Feed service rectifies this by providing daily or weekly changefiles.
+To use the Data Feed:
+1. Get the current snapshot.
+2. Get all changefiles starting with latest timestamp just before the snapshot date.
+3. Apply the changefiles to the snapshot in date order from oldest to newest.
+
+The Data Feed service requires an API key in order to access the changefiles.  See the [data feed page](https://unpaywall.org/products/data-feed)
+for more information on obtaining a key.
+
+On first run, this telescope tries to pull the snapshot on the telescope's start date.  Subsequent scheduled runs will download the daily changefile from __TWO DAYS PRIOR__ to the scheduled execution date to update the dataset.  You should set the scheduled_interval to "@daily" or some other equivalent interval which that results in scheduled runs on a daily basis. The telescope attempts to catch up on missed scheduled runs in case of interruption.
+
+The reason that the changefile applied is from two days prior to each scheduled executin date is so that we can guarantee data integrity after applying the snapshot.
+
```

Comment:
It would be good to put an example the required Airflow Connection in the documentation, i.e. the key name and the connection URI.